apps/full-site-editing/full-site-editing-plugin/full-site-editing-plugin.php

```diff
@@ -40,6 +40,42 @@
 // Always include these helper files for dotcom FSE.
 require_once __DIR__ . '/dotcom-fse/helpers.php';
 
+/**
+ * Load Core Site Editor.
+ */
+function load_core_site_editor() {
+	$options = get_option( 'gutenberg-experiments' );
+	// Initialize options array if needed.
```

Comment:
Having comments be preceded by an empty line helps them stand out more and makes the code a little easier to read. I thought there were WP docs standard around that but I couldn't find any, so I think it's only "wpcs official" :)

Reply:
True story!  LOL, I wasn't expecting anyone to look at it this quickly and had not gotten around to adding the spaces yet.  No Idea why I don't just add them early in the first place.  🤷‍♀ 
Updated!